### ISSUE TITLE
texture bug 

### ISSUE THREAD

**Issue by jochuan** [TimelessModdingTeam/JurassiCraft2#9]
my graphics card doesn't support textures that aren't a power of two, so this make some dinosaurs to appear without texture, i asked bobmowzie to fix my bug and he said that the future updates will have textures that will work for me and for some people that have the same bug. so i like to know if the version 2.0 will fix that.

![635715296456108974](https://cloud.githubusercontent.com/assets/11268912/8508268/83a687e6-2240-11e5-92c7-134a09a0eda3.png)
![635715296665280198](https://cloud.githubusercontent.com/assets/11268912/8508270/8e16e63a-2240-11e5-93fd-dae8e9016c8a.png)
![635715297041679323](https://cloud.githubusercontent.com/assets/11268912/8508272/9b4ad780-2240-11e5-96e9-cb7f703e1b60.png)
![635715297202174409](https://cloud.githubusercontent.com/assets/11268912/8508274/a3545866-2240-11e5-9744-793ea798bbaf.png)

**Comment by BobMowzie** [comment]
I've been complaining to them about this, but I'm not sure that the modelers have been listening.

This is an issue, everyone. Don't ignore it anymore, or fixing it will be a bigger and bigger task.

**Comment by PCAwesomeness** [comment]
Wait, why on the JC2 repository?

**Comment by BobMowzie** [comment]
He wants to know if version 2.0 will fix the bug. As far as I know, the bug is now in EVERY dinosaur instead. So here it stays.

**Comment by PCAwesomeness** [comment]
@BobMowzie 

Hmm, strange.

I'd be surprised if it affected everyone, so, let's try and fix it.

**Comment by BobMowzie** [comment]
Not really strange. The asset team hasn't been paying attention to texture dimensions.

**Comment by PCAwesomeness** [comment]
@BobMowzie 

Well, take it upon yourself to fix them! :)

**Comment by BobMowzie** [comment]
I need to work on things that fewer other people can help with, such as animations.

**Comment by PCAwesomeness** [comment]
@BobMowzie 

Wait, didn't Gegy take that job?

**Comment by BobMowzie** [comment]
No, he's been doing repeatable animations while I've been finishing the Mowzie's Mobs update. I have to make more tricky gaits and frame by frame animations. Once I shift my time back to Jurassicraft, I wouldn't want to spend it all repairing texture sizes.

**Comment by PCAwesomeness** [comment]
@BobMowzie 

OK.

**Comment by jochuan** [comment]
thanks finally i can get a little hope of fixing this bug
thank you very much Bob Mowzie and all the jurassicraft team

**Comment by gegy1000** [comment]
We've told off the modelling team for making evil texture dimensions. :P I will make sure there are no bad texture dimensions before release. :)
